app/app.boot.ts

```diff
@@ -1,4 +1,10 @@
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app.modules';
 
-platformBrowserDynamic().bootstrapModule(AppModule);
\ No newline at end of file
+platformBrowserDynamic().bootstrapModule(AppModule);
+
+// import { platformBrowser }    from '@angular/platform-browser';
```

Comment:
According to [this documentation](https://angular.io/docs/ts/latest/cookbook/aot-compiler.html) we want the commented version here but (a) it works without it and (b) I can't wrap my head around how to make this work right now.